src/utils/constants.ts

```diff
@@ -4,6 +4,7 @@ export const SEARCH_RESULTS_PER_PAGE = 10;
 export const MINIMUM_SEARCH_CHARACTERS = 3;
 export const DEBOUNCE_INTERVAL = 500;
 export const MAX_NOTIFICATIONS = 3;
+export const APP_VERSION = '1';
```

Comment:
Vi burde vel også ha en beskrivelse et sted om hvordan/når vi bruker og oppdaterer denne? Noen av oss glemmer for lett :-)

Reply:
enig - jeg putter det inn i confluence 👍